app/controllers/admin/events_controller.rb

```diff
@@ -74,7 +74,7 @@ def set_event
 
   def event_params
     params.require(:event).permit(
-      :name, :slug, :date_and_time, :begins_at, :ends_at, :description, :info, :schedule, :venue_id, :external_url,
+      :remote, :name, :slug, :date_and_time, :begins_at, :ends_at, :description, :info, :schedule, :venue_id, :external_url,
```

Comment:
@jhackett1 can we rename this field to `virtual` please? So it aligns with the name of the field on `Workshop`.

Reply:
done